### DISCUSSION TITLE
rxjs 7: What does it mean for Akita?

### DISCUSSION THREAD

**Discussion by tony** [salesforce/akita#669] [Ideas]
rxjs 7 was released recently:

- Breaking Changes 6 -> 7: https://rxjs-dev.firebaseapp.com/deprecations/breaking-changes

Blog posts on rxjs 7:
- https://indepth.dev/posts/1353/the-state-of-rxjs-rxjs-7-and-beyond
- https://labs.thisdot.co/blog/rxjs-7-and-beyond-what-to-expect-and-prepare-for

Thoughts? Ideas on how this could impact akita?

I'm not as familiar with rxjs. :)

Could [`firstValueFrom`](https://rxjs-dev.firebaseapp.com/api/index/function/firstValueFrom) become an alternative to `.pipe(take(1)))`?

**Comment by NetanelBasal** [comment]
We'll need to wait for Angular.

**Reply by tony** [reply]
@NetanelBasal What is the relationship between RXJS, akita, and angular?

(I'm not familiar with angular either)

**Comment by eduardoRoth** [comment]
@NetanelBasal now that Angular has supported RxJS 7 (opt-in), can we expect a new release for AkitaJS? Thanks for this amazing library!

**Reply by NetanelBasal** [reply]
You can use it with the latest version.
